src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonOutputFormatter.cs

```diff
@@ -25,16 +24,11 @@ public class JsonOutputFormatter : TextOutputFormatter
         // the serializer and invalidate it when the settings change.
         private JsonSerializer _serializer;
 
-        public JsonOutputFormatter()
-            : this(SerializerSettingsProvider.CreateSerializerSettings(), ArrayPool<char>.Shared)
-        {
-        }
-
-        public JsonOutputFormatter(JsonSerializerSettings serializerSettings)
-            : this(serializerSettings, ArrayPool<char>.Shared)
-        {
-        }
```

Comment:
Users trying to do per-controller specific formatters was fairly common in Web API. In this scenario users would have to new up both input and output formatters. Removing this probably does not give the best experience IMO.